main.go

```diff
@@ -58,15 +62,14 @@ func main() {
 func messageCreate(s *discordgo.Session, m *discordgo.MessageCreate) {
 
 	// Ignore all messages created by the bot itself
-	// This isn't required in this specific example but it's a good practice.
 	if m.Author.ID == s.State.User.ID {
 		return
 	}
+
 	// If the message is "ping" reply with "Pong!"
 	toBino := strings.Contains(m.Content, "bino")
 	if toBino {
 		s.ChannelMessageSend(m.ChannelID, "E cilada bino!!!")
```

Comment:
who is `s`?

Reply:
`s` stands for Session. I changed the param name to make it simpler.